aicsimageprocessing/bin/make_atlas.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import logging
+import os
+import sys
+import traceback
+
+from aicsimageprocessing import get_module_version, textureAtlas
+from aicsimageio import AICSImage
+
+###############################################################################
+
+log = logging.getLogger()
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(asctime)s - %(name)s - %(lineno)3d][%(levelname)s] %(message)s",
+)
+
+###############################################################################
+
+
+class Args(argparse.Namespace):
+    def __init__(self):
+        # Arguments that could be passed in through the command line
+        self.debug = False
+        #
+        self.__parse()
+
+    def __parse(self):
+        p = argparse.ArgumentParser(
+            prog="make_atlas",
+            description="Make a volume-viewer texture atlas from a ome-tiff",
+        )
+        p.add_argument(
+            "-v",
+            "--version",
+            action="version",
+            version="%(prog)s " + get_module_version(),
+        )
+
+        p.add_argument(
+            "--debug", action="store_true", dest="debug", help=argparse.SUPPRESS
+        )
+
+        p.add_argument("infile", type=str, help="input zstack")
+        p.add_argument("outdir", type=str, help="output directory")
+
+        p.parse_args(namespace=self)
+
+
+###############################################################################
+
+
+def main():
+    try:
+        args = Args()
+
+        dbg = args.debug
+        image = AICSImage(args.infile)
+        # preload for performance
+        image.data
```

Comment:
eep! I found what a huge difference this makes. Like, a LOT.  Looking fwd to some kind of fix in aicsimageio in the near future.

Reply:
Resolving as "this is best practice"